x-pack/plugin/spatial/src/main/java/org/elasticsearch/xpack/spatial/SpatialPlugin.java

```diff
@@ -110,13 +122,49 @@ public void registerGeoShapeCentroidAggregator(ValuesSourceRegistry.Builder buil
             });
     }
 
-    public static void registerValueCountAggregator(ValuesSourceRegistry.Builder builder) {
+    public void registerGeoShapeGridAggregators(ValuesSourceRegistry.Builder builder) {
+        builder.register(GeoHashGridAggregationBuilder.NAME, GeoShapeValuesSourceType.instance(),
+            (GeoGridAggregatorSupplier) (name, factories, valuesSource, precision, geoBoundingBox, requiredSize, shardSize,
+                                         aggregationContext, parent, metadata) -> {
+                if (getLicenseState().isAllowed(XPackLicenseState.Feature.SPATIAL_GEO_CENTROID)) {
+                    final GeoGridTiler tiler;
```

Comment:
Should be `SPATIAL_GEO_GRID`?

Reply:
yes! good catch. I was also missing unit tests for this in SpatialPluginTests. added and fixed!